specification/cognitiveservices/Language.AnalyzeConversations-authoring/routes.tsp

```diff
@@ -849,4 +849,81 @@ interface ConversationalAnalysisAuthoring {
     {},
     ErrorResponse
   >;
+
+  /**
+   * Gets the details of an exported model.
+   */
+  @added(Versions.v2024_11_15_preview)
+  getExportedModel is languageOperations.ResourceRead<ConversationalAnalysisAuthoringExportedTrainedModel>;
+  /**
+   * Lists the exported models belonging to a project.
+   */
+  @added(Versions.v2024_11_15_preview)
+  listExportedModels is languageOperations.ResourceList<
+    ConversationalAnalysisAuthoringExportedTrainedModel,
+    ListQueryParametersTrait<StandardListQueryParameters>
+  >;
+  /**
+   * Deletes an existing exported model.
+   */
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @route("/authoring/analyze-conversations/projects/{projectName}/exported-models/{exportedModelName}")
+  @delete
+  @pollingOperation(ConversationalAnalysisAuthoring.getExportedModelJobStatus)
+  @added(Versions.v2024_11_15_preview)
+  deleteExportedModel is Foundations.LongRunningOperation<
```

Comment:
Why is this custom operation needed?

Reply:
Thank you for your comment! Could you please provide more details on this question? The LRO API is already part of our service. Are there any specific modifications we should make to this operation?

Reply:
Yes, you should use LongRunningResourceDelete unless you have a valid reason not to.

Reply:
Our original swagger LRO's 202 response did not include OperationState and OperationId, which will be added if we use the LongRunningResourceDelete here. So I chose to use lower level template for the implementation here.